lib/handlers/pools/provider-migration/v3/pool-provider-traffic-switch-configuration.ts

```diff
@@ -0,0 +1,4 @@
+export const POOL_PROVIDER_TRAFFIC_SWITCH_CONFIGURATION = {
+  switchPercentage: 0.0,
```

Comment:
these should proabbly be secrets/flags so we can unroll them faster if we accidentally push something we do not want

Reply:
I agree, we can probably fast follow once we ship something this week. Since the traffic is at 0% percent, we can fast follow next week if deemed needed.

Reply:
perfect, can we add a jira ticket for this so it does not get forgotten please?

Reply:
@cgkol  ack, added ticket ROUTE-26 into the backlog.